timed/tracking/tests/test_report.py

```diff
@@ -137,6 +137,91 @@ def test_report_list_filter_reviewer(auth_client):
     assert json['data'][0]['id'] == str(report.id)
 
 
+def test_report_list_filter_editable_owner(auth_client):
+    user = auth_client.user
+    report = ReportFactory.create(user=user)
+    ReportFactory.create()
+
+    url = reverse('report-list')
+
+    response = auth_client.get(url, data={'editable': 1})
+    assert response.status_code == status.HTTP_200_OK
+    json = response.json()
+    assert len(json['data']) == 1
+    assert json['data'][0]['id'] == str(report.id)
+
+
+def test_report_list_filter_not_editable_owner(auth_client):
+    user = auth_client.user
+    ReportFactory.create(user=user)
+    report = ReportFactory.create()
```

Comment:
Is it guaranteed that this will create a report with *another* user than "our" user?

Reply:
Yes... because when no user is assigned in a factory it will always create a new one.